test/react-native/features/fixtures/expected_breadcrumbs/HomeReadyNavigation.json

```
@@ -1,6 +1,6 @@
 {
     "type": "navigation",
-    "name": "React Navigation onReady",
+    "name": "React( Native)? Navigation (onReady|componentDidAppear)",
```

Is it worth making these two separate files or matching against either message in its entirety? e.g.

```suggestion
    "name": "(React Native Navigation componentDidAppear|React Navigation onReady)",
```